extensions-core/lookups-cached-global/src/test/java/org/apache/druid/server/lookup/namespace/cache/CacheSchedulerTest.java

```diff
@@ -412,6 +420,62 @@ public void testEntryCloseForgotten() throws InterruptedException
     Assert.assertEquals(0, scheduler.getActiveEntries());
   }
 
+  @Test(timeout = 60_000L)
+  public void testSimpleSubmissionSuccessWithWait() throws InterruptedException
+  {
+    UriExtractionNamespace namespace = new UriExtractionNamespace(
+        tmpFile.toURI(),
+        null, null,
+        new UriExtractionNamespace.ObjectMapperFlatDataParser(
+            UriExtractionNamespaceTest.registerTypes(new ObjectMapper())
+        ),
+        new Period(0),
+        null,
+        null
+    );
+    CacheScheduler.Entry entry = scheduler.scheduleAndWait(namespace, 10_000L);
+    waitFor(entry);
+    Assert.assertEquals(VALUE, entry.getCache().get(KEY));
+  }
+
+
+  @Test(timeout = 20_000L)
+  public void testSimpleSubmissionFailureWithWait() throws InterruptedException
```

Comment:
Do we need to add an expected exception for this test or an assertThrows?
Currently, it just seems to be scheduling a lookup extraction but not asserting anything.

Reply:
The timeout for the test is the assertion. The `scheduleAndWait` waits for 40s before the fix is done - which leads to a failure because the timeout is 20s. After the fix, the test stops failing